ACE/ace/Auto_Ptr.h

```diff
@@ -68,9 +68,14 @@ class ACE_Auto_Basic_Ptr
 
 ACE_END_VERSIONED_NAMESPACE_DECL
 
+// NOTE: ACE7 and TAO3 require C++11 support or newer; jwillemsen commented on Jun 1, 2021
+#if defined (ACE_LACKS_AUTO_PTR)
+#  undef ACE_LACKS_AUTO_PTR
+#endif
+
```

Comment:
You mean I should remove this lines?

Reply:
Just don't change that file at this moment, first get rid of any auto_ptr usage in ACE/TAO

Reply:
done